docs/docs/basic-hardware-software-requirements.md

```diff
@@ -14,6 +14,9 @@ Find the latest hardware and software requirements for building with Gatsby. Not
 
 This will be variable depending on the size of your site. Gatsby sites have been known to work from 500mb to 1GB of RAM.
 
+## Platform specific issues
```

Comment:
Let's move this heading to a `h3` and under the "Operating Systems" section since this is the most relevant place. You'll also need to add the note mentioned in bullet point 1 here: https://github.com/gatsbyjs/gatsby/issues/36000

Currently it only mentions 2)

Reply:
Done, Check if that meets the requirement. 🙂